src/main/java/uk/nhs/digital/nhsconnect/lab/results/configuration/AmqpConfiguration.java

```diff
@@ -0,0 +1,102 @@
+package uk.nhs.digital.nhsconnect.lab.results.configuration;
+
+
+import org.apache.qpid.jms.JmsConnectionFactory;
+import org.apache.qpid.jms.JmsDestination;
+import org.apache.qpid.jms.message.JmsMessageSupport;
+import org.apache.qpid.jms.policy.JmsDefaultDeserializationPolicy;
+import org.apache.qpid.jms.policy.JmsRedeliveryPolicy;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jms.support.converter.MappingJackson2MessageConverter;
+import org.springframework.jms.support.converter.MessageConverter;
+import org.springframework.util.StringUtils;
+
+import javax.jms.ConnectionFactory;
+
+
+@Configuration
+@ConditionalOnMissingBean(ConnectionFactory.class)
+public class AmqpConfiguration {
+
+    @Bean
+    public MessageConverter jsonMessageConverter() {
+        return new MappingJackson2MessageConverter();
+    }
+
+    @Bean
+    public JmsConnectionFactory jmsConnectionFactory(AmqpProperties properties) {
+        JmsConnectionFactory factory = new JmsConnectionFactory();
+
+        factory.setRemoteURI(properties.getBrokers());
+
+        if (!StringUtils.isEmpty(properties.getUsername())) {
```

Comment:
Spring's `SpringUtils.isEmpty` is deprecated so we should switch to Apache Common's `SpringUtils.isBlank` method because it also checks if a string is whitespace as well as if the string is null or empty ("")

```suggestion
        if (!StringUtils.isBlank(properties.getUsername())) {
```

Reply:
Thanks! Is it better now? 59709bd6b934757511a4e6efa3a0e921c7420445?

Reply:
I'd remove the `org.springframework.util.` part on lines 62 and 66 but other than that, it looks good

Reply:
That's the thing, I have tried to, but because hasLength function comes from another library, I didn't find the way to do it differently